app/controllers/api/connect/v3/systems/products_controller.rb

```diff
@@ -12,7 +12,13 @@ def activate
   end
 
   def show
-    if @system.products.include? @product
+    if @product.identifier.casecmp('sles').zero?
```

Comment:
Is there no positive test here we can do? It feels inefficient to say "if it's not sles maybe it's sle-micro".